src/constants/ChatConstants.js

```diff
@@ -11,7 +11,8 @@ export default {
     RECEIVE_RAW_MESSAGES: null,
     STORE_HISTORY_MESSAGE: null,
     THEME_CHANGED:null,
-    SEARCH_MODE: null
+    SEARCH_MODE: null,
```

Comment:
as i see this one line is not belongs to this issue.could you please check it agin?